### ISSUE TITLE
issue in domain generator when creating an associated application

### ISSUE THREAD

**Issue by FabienDehopre** [angular-architects/nx-ddd-plugin#134]
In the latest nx workspace, when you try to create a domain with an associated application (`--addApp=true`), the generator fail with the following error message: "Cannot read properties of undefined (reading 'replace')".
In fact, the issue comes from the fact that the generator tries to get the workspace npm scope from the nx.json file (using the `readWorkspaceConfiguration` function).
Unfortunately, the `npmScope` property on the nx.json file is now deprecated (see official doc: https://nx.dev/packages/devkit/documents/Workspace#npmscope). 
To get the workspace npm scope, you need to use the `getNpmScope` function instead.

**Comment by L-X-T** [comment]
Thank you for the report @FabienDehopre, I've encountered a similar problem which is also due to the reading of the npmScope. I'll inform @manfredsteyer.

**Comment by L-X-T** [comment]
@scsete no, I'm very sorry: currently no advice. I'll ask @manfredsteyer once more. I know he's been rather busy recently.
